extensions/wikia/AnalyticsEngine/AnalyticsProviderIVW2.php

```diff
@@ -38,15 +38,17 @@ function trackEvent($event, $eventDetails = array()) {
 					"sv" => "ke"      // FRABO-Tag deaktiviert
 				];
 
-				$ivwScriptTag = '<script src="https://script.ioam.de/iam.js"></script>';
-				$ivwScriptTag .= '<script>iom.c(' . json_encode($iamData) . ', 2);</script>';
-				$ivwScriptTagEscaped = json_encode($ivwScriptTag);
+				$ivwScriptTag = json_encode('<script src="https://script.ioam.de/iam.js"></script>');
+				$ivwTrackTag = json_encode('<script>iom.c(' . json_encode($iamData) . ', 2);</script>');
```

Comment:
Can you contact IVW if it hurts them if we execute it twice? However, those are different data, aren't they?